tests/NuGet.IndexingTests/TestSupport/MockSearcherManager.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Indexing;
+
+namespace NuGet.IndexingTests.TestSupport
+{
+    public class MockSearcherManager : NuGetSearcherManager
+    {
+        private string _indexName;
+
+        public MockSearcherManager(string indexName) : base(indexName, new MockLogger(), null, null, null)
```

Comment:
I would prefer not to touch the existing code without a production need, especially just to enable test scenarios.

However, I'm willing to change it if the majority opinion is different.